api/dfs.js

```diff
@@ -43,19 +43,24 @@ async function createProject(accountId, name) {
  * @param {string} projectFile
  * @returns {Promise}
  */
+
 async function uploadProject(
   accountId,
   projectName,
   projectFile,
-  uploadMessage
+  uploadMessage,
+  platformVersion = ''
```

Comment:
We don't need a default value here anymore

Reply:
Addressed in https://github.com/HubSpot/cli-lib/pull/12/commits/760274c2778a1f3b3f3d9ee3a815e217ee24c0e4